### ISSUE TITLE
Compare tab - edits

### ISSUE THREAD

**Issue by acollier-oxandale** [MazamaScience/AirSensorDataViewer#47]
It would be helpful to make the labeling a little more consistent on the timeseries:

1. Title – “Sensor/Reference Monitor comparison -- PurpleAir [NAME] is x.x km from Reference Monitor: [Station Name] (located in air district: [airDistrict]) 

2. Legend – change ‘PA hourly’ to “Sensor hourly” and “PWFSL” to “Reference Monitor hourly”, also add light grey series and appropriate label to legend

(version 1.0.1)

**Comment by jonathancallahan** [comment]
I believe that the suggestions below will make for strings too long and will make for 2-line/cluttered/ugly graphics.

1. The string "(located in air district: [airDistrict])" will significantly increase the length of the title and is likely to cause other issues like automatic font scaling, etc.
2. I don't believe it is appropriate to sacrifice space allocated to the data to increase the verbiage in a legend title that will never change. Once people figure out what the colors are, the legend becomes just a reminder.

Comparing string lengths:

```
12345678901 -- 11 character max
* PA raw
* PA hourly
* PWFSL

12345678901234567890123456 -- 26 character max
* Sensor raw
* Sensor hourly
* Reference monitor hourly
```

Also, thinking about the future:

What if there is another kind of sensor and you want to plot both "PA hourly" and "Type2 hourly"? It's nice to differentiate.

**Comment by acollier-oxandale** [comment]
We agree regarding limiting the string length, instead of the original request, could "PWFSL" in the timeseries plot title and in the legend be changed to "Monitor"? Then it might be helpful to add some static text below the plot noting that the PWFSL package is used to access to reference data and the source of the reference data.

**Comment by jonathancallahan** [comment]
Resolved in AirSensor 1.0.5
